src/main/java/org/mule/extensions/jms/api/destination/TopicConsumer.java

```diff
@@ -52,7 +53,8 @@
   @Optional(defaultValue = "false")
   @Expression(NOT_SUPPORTED)
   @Summary("Only for JMS 2.0: Allows the processing of messages from a topic subscription by multiple threads, connections or JVMs")
-  private boolean isShared;
+  @Alias("isShared")
+  private boolean shared;
```

Comment:
i think you cannot do this. This will cause the ObjectType to say the field is called `isShared` and is going to appear like that in the DW scafolding that tooling generates. But `setIsShared` doesn't exists. What does exists is `setShared` and thus `kaboom`